Makefile

```diff
@@ -24,43 +22,25 @@ endif
 
 ## Functions
 get_github_latest_version = $$(docker run --rm easy_infra:latest /bin/bash -c "curl -s https://api.github.com/repos/$(1)/releases/latest | jq -r '.tag_name'")
-update_dockerfile_package = ./update.sh --package=$(1) --version=$(2)
-update_dockerfile_repo    = ./update.sh --repo=$(1) --version=$(2)
+update_dockerfile_package = ./update_components.sh --package=$(1) --version=$(2)
+update_dockerfile_repo    = ./update_components.sh --repo=$(1) --version=$(2)
 
 
 ## Rules
 .PHONY: all
 all: update build
 
 .PHONY: update
```

Comment:
why only shorten dependencies to deps.  You don't shorten requirements to reqs.
the whole word is more clear

Reply:
Sure, done.